index.html

```diff
@@ -2035,9 +2030,37 @@ <h3>Securing Mechanisms</h3>
 mutually exclusive.
         </p>
         <p>
-<a>Verifiable credentials</a> and <a>verifiable presentations</a> MUST be secured by at least
-one securing method.
+<a>Verifiable credentials</a> and <a>verifiable presentations</a> MUST be
+secured by at least one securing mechanism.
+        </p>
+        <p>
+Securing mechanism specifications:
         </p>
+
+        <ul>
+          <li>
+MUST document normative algorithms that can be used to secure and verify
+<a>verifiable credentials</a> and <a>verifiable presentations</a>.
+          </li>
```

Comment:
I don't like this text.

Whats the intention here?

Reply:
> My challenge is this seems to create a normative requirement in a foreign spec, to force that spec to "understand" credentials and presentations.

Right, I can see how we don't want to convey that the securing specs need to understand VCs and VPs in order to secure them; they shouldn't. However, "securing bytes" isn't necessarily the right thing to say either (because Data Integrity does more than (on interpretation of) "secure bytes").

I'll try to think of a way to reword this so that it doesn't imply that the securing specifications need to understand VCs/VPs.

Reply:
If they shouldn't understand VCs and VPs, then how can running the securing mechanism verification algorithm verify the authorship of a VC or VP?

Reply:
A verifiable credential is "a tamper-evident credential that has authorship that can be cryptographically verified.". I thought the securing mechanism would do that?

Reply:
I raised issue #1386 to track @awoie's concern.

I made an attempt at addressing @OR13's concern in commit 5e767d9f44f341ccd423b6a2e99268a493db3e9d.